features/error/viewing_errors/viewing_forbidden_error.feature

```diff
@@ -0,0 +1,13 @@
+@viewing_errors
+Feature: Viewing forbidden page
+    In order to keep a good navigation
+    As a visitor
+    I need to be able to view a well forbidden page when page I requested is forbidden
+
+    Background:
+        Given the store operates on a single channel in "United States"
+
+    @ui
+    Scenario: Viewing forbidden page
+        When I am on forbidden page
+        Then I should see the title "Unexpected error occurred."
```

Comment:
Can we change this behavior, so 403 will explicitly say, that access is forbidden?

Reply:
I totally agree, this is a little bug on sylius.